src/mediapipe_calculators/modelapiovmsadapter.cc

```diff
@@ -53,6 +53,8 @@ using std::endl;
 #define CREATE_GUARD(GUARD_NAME, CAPI_TYPE, CAPI_PTR) \
     std::unique_ptr<CAPI_TYPE, decltype(&(CAPI_TYPE##Delete))> GUARD_NAME(CAPI_PTR, &(CAPI_TYPE##Delete));
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
This is to stay because compiler complains when we build with MEDIAPIPE_DISABLED=1?

Reply:
Yes.